docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.9'
+
+services:
+  invention:
+    container_name: invention
+    image: invention/airbnb:1.0.0
+    build: .
+    ports:
+      - '${APP_PORT}:${APP_PORT}'
```

Comment:
I think that in EXPOSE in Dockerfile, you are writing 3000, why here it depends on the APP PORT?